pkg/util/log/eventpb/telemetry.proto

```diff
@@ -256,7 +256,7 @@ message ChangefeedEmittedBytes {
   int64 job_id = 2 [(gogoproto.jsontag) = ",omitempty", (gogoproto.moretags) =  "redact:\"nonsensitive\""];
 
   // The number of bytes emitted.
-  int32 emitted_bytes = 3 [(gogoproto.jsontag) = ",omitempty", (gogoproto.moretags) = "redact:\"nonsensitive\""];
+  int64 emitted_bytes = 3 [(gogoproto.jsontag) = ",omitempty", (gogoproto.moretags) = "redact:\"nonsensitive\""];
```

Comment:
Non-rhetorical question--is this in general a backwards-compatible change to make to a protobuf?

Reply:
That's a good point. I'll ask internally on Slack. I know you can add new events to the telemetry protobuf, but I'm not sure about changing one like so.